platform/.gitignore

```diff
@@ -71,6 +71,7 @@ typings/
 # dotenv environment variables file
 .env
 .env.test
+.env.local
```

Comment:
I was working on the platform repo and added a new env file. This simply ignores the env file so it's not added to the repo